torpaste.py

```diff
@@ -1,4 +1,4 @@
-#!bin/python
```

Comment:
Well, whatever the case (either you use the hasbang or not), the "defacto" hashbang line is `which env` python<version>. That's due to python versioning madness and distro binary naming. Both virtualenv and popular distros (debian/fedora/arch/...) support env.